Dockerfile

```diff
@@ -1,56 +1,32 @@
-# syntax=docker/dockerfile:1.2
+FROM bitnami/minideb:bullseye AS setup
 
-# build base that sets up common dependencies of the build
-FROM rust:alpine as build-base
+RUN apt-get update && apt-get install -y \
```

Comment:
I think you should probably use `install-packages` (https://github.com/bitnami/minideb#why-use-minideb) for this to prevent excess metadata appearing in the docker images